mautic/mautic
# Comment improvements

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | kinda
| New feature?                           | a little
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | /
| Related developer documentation PR URL | /changes -->
| Issue(s) addressed                     | /




#### Description:

This is a remnant of a plugin our team has built. We set our PHPSTAN rules high and some method annotations with param and return types were messing with it as they were wrong. So we fixed those.

The only production code change is in the `togglePublishStatusAction` method where the ID now doesn't have to be an integer but also UUID which we are using on the new tables instead of auto-increment IDs as it has several advantages.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Ensure you have some email, form, page for example.
3. On the list page there is a switch to un/publish the entity.
4. Make sure you can unpublish and publish the entity with this switch.
